bool    onOFF
float32 heading
float32 heading_demand
float32 error
float32 speed
float32 ddelta
float32 delta
float32 dT2
float32 dT3
float32 T2
float32 T3
int32   thruster2
int32   thruster3
float32 delta_gain
float32 thruster_gain
float32 thruster_penalty
float32 Np
float32 Nc
float32 delta_t
float32 calc_time
int32   km